client/src/components/ProjectWizard/WizardPages/ProjectSummary/ProjectInfoList.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { createUseStyles } from "react-jss";
+
+const useStyles = createUseStyles({
+  projectInfoDetailsSubContainer: {
+    display: "flex",
+    alignItems: "baseline",
+    width: "50%",
+    paddingBottom: "1.1em"
+  },
+  projectInfoCategory: {
+    fontFamily: "Oswald",
+    fontWeight: "500",
+    fontSize: "12px",
+    textTransform: "uppercase",
+    color: "rgba(15, 41, 64, .5)",
+    marginRight: "17px"
+  },
+  projectInfoDetails: {
+    color: "#0F2940",
+    fontFamily: "Calibri Bold",
+    fontSize: "16px"
+  },
+  AINValuesContainer: {
+    display: "flex",
+    flexWrap: "wrap",
+    flexDirection: "row"
+  },
```

Comment:
Looks good! Maybe consider doing justifyContent: "space-between" in the AINValuesContainer rather than minWidth in AINValues. Also maybe add gap: "5px" or some value like that so space out the AIN values.

Reply:
Thanks for highlighting this thing. I missed that part. Certainly, I will work on that and will fix it.